### ISSUE TITLE
Accessibility issue, running under Wayland doesn't respect font scaling

### ISSUE THREAD

**Issue by sillyspiel** [strawberrymusicplayer/strawberry#1456]
With larger fonts set under Accessibility in Gnome, Strawberry's font sizes do not change. Issue doesn't exist using X11 session. Can be worked around by using flatpak version, opening Flatseal and turning off "Wayland windowing system", leaving only "X11 windowing system" and "Fallback to X11 windowing system". Should note that flatpak version running on Plasma also needs this workaround (I think? having trouble remembering right now and unable to test atm).

**Comment by jonaski** [comment]
Other Qt 6 based programs respects this Gnome accessibility?
Fill in the issue template: OS, Qt version, strawberry version. Everything is missing in this bug report.

**Comment by sillyspiel** [comment]
> Other Qt 6 based programs respects this Gnome accessibility setting? Fill in the issue template: OS, Qt version, strawberry version. Everything is missing in this bug report.

I'm not able (or willing) to test this all out again as I'm currently back on Linux Mint. I should have filed this bug report while I still had the VM/bare metal installs running, but alas I got distracted. Please just look into it if you have the time or willingness. I really like this program by the way and it's what I prefer for music players. For what it's worth, issue occured on openSUSE Tumbleweed, openSUSE Aeon, Fedora 40 Workstation (Gnome), and Fedora 40 KDE. On openSUSE, it is definitely version 1.0.23, same with Fedora. On openSUSE, Help > About says it is using Qt version 6.7.1. On Fedora, it says it is using 5.15.13, and on flatpak 6.6.3

**Comment by jonaski** [comment]
To be honest I do not have time to boot up every desktop and test different settings, I doubt this is a bug in strawberry itself, but more likely a general Qt/Gnome related issue. Except for the lyrics we do not hard-code font sizes, I do not know if the Gnome accessibility settings are integrated properly with Qt applications.

**Comment by sillyspiel** [comment]
>I do not know if the Gnome accessibility settings are integrated properly with Qt applications.

Further research has me thinking this is probably the actual issue. But thank you for looking into it :)
